### PR TITLE
Pytest `test_stylize_fail` removed xfail

### PR DESCRIPTION
**Pull request by abhishekpratapa**
- Added error checking for invalid styles types
    - Empty Lists `[]`
    - Indices that are out of the proper range from `0` to `num_styles - 1`

### CODE REVIEW

File: src/toolkits/style_transfer/style_transfer.cpp [hunk 1]
```diff
@@ -463,9 +463,13 @@ gl_sframe style_transfer::predict(variant_type data,
         break;
       case flex_type_enum::VECTOR:
         style_idx = std::move(flex_style_idx.get<flex_vec>());
+        if(style_idx.size() == 0)
```
Comment (nickjong) [inline src/toolkits/style_transfer/style_transfer.cpp]: Nit: use clang-format for consistent formatting, e.g. leave a space between `if` and opening parenthesis

For STL containers, prefer `.empty()` to `.size() == 0`

File: src/toolkits/style_transfer/style_transfer.cpp [hunk 2]
```diff
@@ -521,6 +525,9 @@ void style_transfer::perform_predict(gl_sarray data, gl_sframe_writer& result,
 
   // looping through all of the style indices
   for (size_t i : style_idx) {
+    // check whether the style indicies are valid
```
Comment (nickjong) [inline src/toolkits/style_transfer/style_transfer.cpp]: Check spelling: "indices"

File: src/toolkits/style_transfer/style_transfer.cpp (hunk 1 above)
Comment (abhishekpratapa) [reply src/toolkits/style_transfer/style_transfer.cpp]: Solid!

File: src/toolkits/style_transfer/style_transfer.cpp (hunk 2 above)
Comment (abhishekpratapa) [reply src/toolkits/style_transfer/style_transfer.cpp]: I don't know how I passed 2nd grade. LMAO. Thanks!

Comment (nickjong) [reply src/toolkits/style_transfer/style_transfer.cpp]: Don't worry, no 2nd grader could do what you do!

**Pull request merged**